### PR TITLE
[NCL-2314] Added ID to response from creating product

### PR DESCRIPTION
**Pull request by stanlyDoge**

### CODE REVIEW

**Comment by stanlyDoge** [comment]
retest this please

File: reports-model/src/main/java/org/jboss/da/listings/model/rest/SuccessResponse.java [hunk 1]
```diff
@@ -33,6 +33,11 @@
 
     @Getter
     @Setter
+    @XmlElement(required = true, name = "id")
```
Comment (janinko) [inline reports-model/src/main/java/org/jboss/da/listings/model/rest/SuccessResponse.java]: I think it's not required, as you set the field only in one place.

**Comment by janinko** [comment]
retest this please

**Comment by janinko** [comment]
:+1:

**Comment by janinko** [comment]
retest this please

**Comment by stanlyDoge** [comment]
retest this please

**Comment by stanlyDoge** [comment]
retest this please

**Comment by stanlyDoge** [comment]
retest this please

**Comment by stanlyDoge** [comment]
passed on local

**Comment by janinko** [comment]
retest this please

**Pull request merged**